march_hardware/include/march_hardware/IMotionCube.h

```diff
@@ -64,6 +64,8 @@ class IMotionCube : public Slave
   void goToTargetState(IMotionCubeTargetState target_state);
   virtual void goToOperationEnabled();
 
+  void resetIMotionCube();
```

Comment:
The class is already called `IMotionCube` so it would be weird to call this method `resetIMotionCube`, since that is what it is. Furthermore, this would looks redundant when you try to call it: `imotioncube.resetIMotionCube()`.

Reply:
right, good call. Nice review!